package.json

```diff
@@ -46,6 +46,28 @@
     "test": "./node_modules/.bin/karma start ./karma.conf.js",
     "travis": "./node_modules/grunt-cli/bin/grunt ci --verbose --force & npm run test"
   },
-  "dependencies": {},
+  "dependencies": {
```

Comment:
All of these are pinned with a `^` and from what I understand this keeps them within a major version release. Source: https://docs.npmjs.com/misc/semver#caret-ranges-123-025-004

Is within a major version release good enough for this? Do most of these not adhere to semver?